package.yaml

```diff
@@ -1,7 +1,7 @@
 name: github-workflow-commands
 version: 0.0.0.0
 maintainer: Freckle Education
-category: Utils
+category: GitHub
```

Comment:
Wow this is really [the wild west](https://hackage.haskell.org/packages/) huh?

Reply:
Yeah I try to find some balance between "is this category too broad to be useful" or "is this category so narrow it's just clutter." I still find that page listing packages by category to be helpful every once in a while when I'm looking for something, but the Foundation is going to have to apply some sort of top-down organization effort if they ever want it to actually be good.